packages/curve-ui-kit/src/lib/i18n.ts

```diff
@@ -54,40 +37,21 @@ export function parseLocale(locale?: string): { parsedLocale: Locale['value']; p
   }
 }
 
-/**
- * Activates messages data for a dynamically loaded locale.
- *
- * Example:
- * ```ts
- * const messages = await import(`@/locales/en/messages`)
- * dynamicActivate('en', messages)
- * ```
- *
- * Note: The import must be performed by the calling app since the import path
- * is relative to the app's location, not this package. Passing the import path
- * as a parameter (e.g. '@/locales/en/messages') would not work because the path
- * would be resolved relative to this package's directory structure rather than
- * the app's, and thus will not be able to be found.
- *
- * @param locale - The locale identifier (e.g. 'en', 'zh-Hans')
- * @param data - The imported locale messages data
- */
-export function dynamicActivate(locale: string, data: { messages: Record<string, string> }) {
-  i18n.load(locale, data.messages)
-  i18n.activate(locale)
-}
-
 export function updateAppLocale(locale: string) {
   setDayjsLocale(locale as Locale['value'])
+}
 
-  let numbroLang = ''
-  if (locale === 'zh-Hant') numbroLang = 'zh-TW'
-  if (locale === 'zh-Hans') numbroLang = 'zh-CN'
+/**
+ * Placeholder for a translation function, so that we can introduce a real one later.
+ * @param key Key to translate. This is an array if there are variables to interpolate.
+ * @param template Variables to interpolate. They are appended to the keys of the same index.
+ * @returns The original string, or the string with the variables interpolated.
+ */
+export const t = (key: string | readonly string[], ...template: unknown[]) =>
+  Array.isArray(key) ? key.map((k, i) => (i < template.length ? `${k}${template[i]}` : k)).join('') : (key as string)
 
-  if (numbroLang) {
```

Comment:
If we're no longer configuring numbro I wonder if we can just get rid of the package alltogether maybe? Or at least get rid of the top import `import 'numbro/dist/languages.min.js'`. Maybe get rid of Dayjs too...

Reply:
I'll get rid of the languages import. It's still used in 2 files, we can fix that separately
https://github.com/search?q=repo%3Acurvefi%2Fcurve-frontend%20numbro&type=code